model/perf.go

```diff
@@ -272,7 +273,11 @@ func (r *PerformanceResults) Find(options PerfFindOptions) error {
 	}
 
 	r.populated = false
-	err = session.DB(conf.DatabaseName).C(perfResultCollection).Find(search).All(&r.Results)
+	query := session.DB(conf.DatabaseName).C(perfResultCollection).Find(search).Sort("-" + perfCreatedAtKey)
```

Comment:
is there an index for this?

Reply:
There's a created_at index but since this query can be many shapes it may or may not be used